design-library/source/_patterns/04-pages/02-article/article.twig

```diff
@@ -0,0 +1,32 @@
+<div id="page-wrapper" class="page-wrapper">
+  <div id="main-wrapper" class="main-wrapper">
+    <div class="sidebar">
```

Comment:
If these "div" tags are shared across pages, we could consider making them patterns and re-using the patterns on multiple pages.